### ISSUE TITLE
Import from rezeptwelt.de broken

### ISSUE THREAD

**Issue by obi134** [TandoorRecipes/recipes#1561]
### Tandoor Version

1.0.8

### Setup

Docker / Docker-Compose

### Reverse Proxy

No reverse proxy

### Bug description

I tried to import a recipe with a url:
https://www.rezeptwelt.de/hauptgerichte-mit-fleisch-rezepte/hackbaellchen-toskana/j8zsddgt-14210-586301-cfcd2-btx0xims

Generally the import works, but the description is not parsed correctly. I'm getting:

```
{'@type': 'HowToSection', 'name': ['Soße', 'Hackbällchen', 'Beilage'], 'position': ['1', '2', '3'], 'itemListElement': [{'@type': 'HowToStep', 'position': '1', 'name': '- 1 Zwiebe...', 'text': '- 1 Zwiebel, groß geschält und halbiert\n\n2 Knoblauchzehen, geschält\n\nin den "Mixtopf geschlossen" geben\n\nauf 4 sek / Stufe 5 zerkleinern\n\n- 25 g Öl dazu "Mixtopf geschlossen"\n\n2 min / Varoma / Stufe 1 dünsten\n\n- 1 Dose stückige Tomaten zugeben\n\n2 TL Brühpulver,\n\netwas Pfeffer, Salz, Paprika rosenscharf\n\n2 TL ital. Kräuter dazu "Mixtopf geschlossen"\n\nauf 12 min / 100°c / Stufe 2 kochen\n\n- 1 Becher Schmand zugeben\n\n1 Schluck Milch in den "Mixtopf geschlossen"\n\netwas verühren'}, {'@type': 'HowToStep', 'position': '2', 'name': 'Solange di...', 'text': 'Solange die Soße kocht\n\n- 500 g gem. Hackfleisch in eine Schüssel geben.\n\n- mit Salz, Pfeffer und Paprika rosenscharf würzen\n\n- ca 15 kleine Bällchen formen und in eine gefettete Auflaufform setzen.\n\nMit der Soße aus dem "Mixtopf geschlossen" übergießen.\n\nMozarella in Scheiben schneiden und alles damit bedecken\n\nIm Backofen vorgeh. 200°c - 30 - 35 min überbacken!'}, {'@type': 'HowToStep', 'position': '3', 'name': 'Ich esse d...', 'text': 'Ich esse dazu gerne Reis, aber ihr könnt natürlich auch Nudeln oder Kartoffeln dazu essen.\n\nWährend die Auflaufform im Ofen ist, setze ich im "Mixtopf geschlossen" den Reis an.\n\n300g Landkornreis, etwa 1,3 l Wasser mit einer Prise Salz\n\n100°c/ 30 min/ Stufe 2\n\nIst dann perfekt zusammen fertig Lasst es Euch schmecken !'}]}

Imported from https://www.rezeptwelt.de/hauptgerichte-mit-fleisch-rezepte/hackbaellchen-toskana/j8zsddgt-14210-586301-cfcd2-btx0xims
```
![hack](https://user-images.githubusercontent.com/6304830/154818687-a1ad0c0b-ff79-442c-a810-be37defc256d.png)

**Comment by vabene1111** [comment]
very interesting format, not sure if that is standard conform but should be possibel to teach to the parser

**Comment by smilerz** [comment]
If that site isn’t importing correctly please open an issue at https://github.com/hhursev/recipe-scrapers

**Comment by obi134** [comment]
Hi @smilerz and @vabene1111 ,
Thank you for your respone.
I just tried with recipe-scrapers to get the information directly and it worked well:

![recipt_scraper](https://user-images.githubusercontent.com/6304830/154859652-97594ee9-e776-4155-bf91-2bbddcdc28db.png)

So what is the difference?

**Comment by smilerz** [comment]
Is the version of scraper the same?

**Comment by obi134** [comment]
I just downloaded a few minutes ago, it is recipe-scrapers==13.18.0

**Comment by obi134** [comment]
Ahh, Version 13.13.0 added scraper for rezeptwelt.de
This relates also to https://github.com/TandoorRecipes/recipes/issues/1491

**Comment by smilerz** [comment]
> I just downloaded a few minutes ago, it is recipe-scrapers==13.18.0
> I don't know which version of it is included in tandoor 1.0.8

I just lo loked. Support for that site started in 13.13.0 which is later than what is currently in the release. The scraper will be updated at the next release.

**Comment by obi134** [comment]
Perfect, thank you!
